lib/vecmat.gi

```diff
@@ -1473,18 +1473,20 @@ local sf, rep, ind, ind2, row, i,big,l;
   fi;
 
   # this can only happen if not big
-  for i in ind do
-    ConvertToVectorRepNC(matrix[i],sf);
-  od;
+  if not big then
```

Comment:
This seems to be the easiest way to make sure that the `Convert...NC` function is not called if `big=true`. Actually, I did not try to understand or check the code that determined `ind`, because I don't know what exactly this function is supposed to do.

This pull request is only meant as quick and local fix (to make  `AlgebraicExtension` work for me).

I'm happy if someone who knows better this code provides a different fix (and maybe adds some explanations about the meaning of this code).